frontend/src/scenes/surveys/SurveyResponseLimitWidget.tsx

```diff
@@ -0,0 +1,64 @@
+import { IconInfo } from '@posthog/icons'
+import { LemonBanner, Link, Tooltip } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { billingLogic } from 'scenes/billing/billingLogic'
+import { userLogic } from 'scenes/userLogic'
+
+const DEFAULT_SURVEY_RESPONSE_LIMIT = 250
+
+export function SurveyResponseLimitWidget(): JSX.Element | null {
+    const { billing } = useValues(billingLogic)
+    const { user } = useValues(userLogic)
+
+    // Only show for non-admin users
+    if (user?.is_staff || user?.is_impersonated) {
+        return null
+    }
+
+    // Check if we have survey response usage data
+    const surveyResponsesUsage = billing?.usage_summary?.survey_responses
+    if (!surveyResponsesUsage) {
```

Comment:
```suggestion
    if (!surveyResponsesUsage.usage) {
```

if usage is 0 then we can just hide it